rpmbuild/copr_rpmbuild/builders/mock.py

```diff
@@ -42,6 +42,10 @@ def __init__(self, task, sourcedir, resultdir, config):
         self.macros = macros_for_task(task, config)
         self.uniqueext = get_mock_uniqueext()
         self.allow_user_ssh = task.get("allow_user_ssh")
+        self.tags = task.get("tags", [])
+        self.max_fs_size = "140g"
+        if "on_demand_powerful" in self.tags:
+            self.max_fs_size = "280g"
```

Comment:
I'm not against having (some) default, but in 99% this is deployment-specific info.  And 140G is _typically_ quite too much for local reproducibility (the user tries to run the copr-rpmbuild locally to reproduce a build failure, e.g.).